### PR TITLE
BD-1277: Increase exposure of nested objects

### PR DESCRIPTION
**Pull request by bre-fitzgerald**
# Pull Request/Issue Resolution

Closes #**BD-1277**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (__Insert Feature Release Date Here__)
- [ ] No

### CODE REVIEW

File: _docs/_user_guide/data_and_analytics/custom_data/custom_events.md [hunk 1]
```diff
@@ -110,6 +111,145 @@ When making API calls and using the "is blank" filter, a specific custom event p
...
+`r_details.name`: "Mcdonalds"<br>
+`r_details.location.city`: "Montclair"
+{% endtab %}
+{% endtabs %}
+
+{% alert note %} If your event property contains the `[]` or `.` characters, escape them by wrapping the chunk in double-quotes. For instance, `"songs[].album".yearReleased` will match an event with the literal property `"songs[].album"`.  {% endalert %}
+
+##### Segmentation
+
+Use [segment extensions]({{site.baseurl}}/user_guide/engagement_tools/segments/segment_extension/) to segment users based on nested event properties. Segmentation uses the same notation as triggering (described above).
```
Comment (johndinh) [inline _docs/_user_guide/data_and_analytics/custom_data/custom_events.md:241]: Do we need to remind customers that segmentation with nested event properties is in early access?

**Review by johndinh — DISMISSED** [review]
Added two comments for Bre's discretion.  Otherwise, LGTM!

File: _docs/_user_guide/data_and_analytics/custom_data/custom_events.md (hunk 1 above)
Comment (bre-fitzgerald) [reply _docs/_user_guide/data_and_analytics/custom_data/custom_events.md:241]: Makes sense to me—I can add that in 👍

**Review by KellieHawks — CHANGES_REQUESTED** [review]
Looks good! Just missing a description for the two new GA pages. :)

**Review by KellieHawks — APPROVED** [review]
Looks great!

**Pull request merged**